components/input/Input.tsx

```diff
@@ -118,7 +118,7 @@ export interface InputProps
   disabled?: boolean;
   status?: InputStatus;
   bordered?: boolean;
-  [key: `data-${string}`]: string;
+  [key: `data-${string}`]: string | undefined;
```

Comment:
```suggestion
  [key: `data-${string}`]: string | undefined;
```
```suggestion
  [key: `data-${string}`]?: string;
```

Reply:
```ts
[key: `data-${string}`]?: string;
```
is not valid TypeScript code.

https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgJIgA4FcwAUoD2GAzsgN4CwAUMsgCbDFwBGANhHQPwBcyzBBdnBABuarWJg4YLMR5pMOAMpSZxMTT4EodaB3n9BEYRtoBtANYQAnrwAGdaXAC0AEjKSooAOYBfOwC68p4+Gr5AA